lectures/supply_demand_multiple_goods.md

```diff
@@ -422,13 +602,273 @@ $$
 p = \mu^{-1} [\Pi^\top b - \Pi^\top \Pi c]
 $$ (eq:old5pa)
 
-Equating this to the inverse supply curve and solving
+Equating this to the inverse supply curve, letting $q=c$ and solving
 for $c$ gives
 
 $$
 c = [\Pi^\top \Pi + \mu H]^{-1} [ \Pi^\top b - \mu h]
 $$ (eq:old5p)
 
++++
+
+Here we define a class ``ProductionEconomy``.
+
+```{code-cell} ipython3
+class ProductionEconomy:
```

Comment:
The class ``ProductionEconomy`` and its examples are moved directly from the lecture ``supply_demand_heterogeneity``. 

I suggest we delete this class in the lecture ``supply_demand_heterogeneity`` in the following PR:
- https://github.com/QuantEcon/lecture-python-intro/pull/220

Reply:
I agree @shlff